wagtail/admin/mail.py

```diff
@@ -85,30 +101,214 @@ def send_notification(page_revision_id, notification, excluded_user_id):
+                    with override(recipient.wagtail_userprofile.get_preferred_language()):
+                        # Get email subject and content
+                        email_subject = render_to_string(template_set['subject'], context).strip()
+                        email_content = render_to_string(template_set['text'], context).strip()
+
+                    kwargs = {}
+                    if getattr(settings, 'WAGTAILADMIN_NOTIFICATION_USE_HTML', False):
+                        kwargs['html_message'] = render_to_string(template_set['html'], context)
+
+                    # Send email
+                    send_mail(email_subject, email_content, [recipient.email], connection=open_connection, **kwargs)
+                    sent_count += 1
+                except Exception:
+                    logger.exception(
+                        "Failed to send notification email '%s' to %s",
+                        email_subject, recipient.email
+                    )
+
+        return sent_count == len(recipients)
+
+    def __call__(self, notifying_instance, notification, **kwargs):
+        """Send emails corresponding to the notification (eg 'approved') from an instance, notifying_instance"""
+
+        if not self.can_handle(notifying_instance, notification, **kwargs):
+            return False
+
+        recipients = self.get_valid_recipients(notifying_instance, notification, **kwargs)
+
+        if not recipients:
+            return True
+
+        template_set = self.get_template_set(notifying_instance, notification, **kwargs)
+
+        context = self.get_context(notifying_instance, notification, **kwargs)
+
+        return self.send_emails(template_set, context, recipients, **kwargs)
+
+    def approved(self, instance=None, **kwargs):
+        return self(instance, 'approved', **kwargs)
+
+    def rejected(self, instance=None, **kwargs):
+        return self(instance, 'rejected', **kwargs)
+
+    def submitted(self, instance=None, **kwargs):
+        return self(instance, 'submitted', **kwargs)
+
+
+class WorkflowStateNotifier(Notifier):
+    """A Notifier to send updates for WorkflowState events"""
+
+    def __init__(self, valid_notifications):
+        super().__init__({WorkflowState}, valid_notifications)
+
+    def get_recipient_users(self, workflow_state, notification, **kwargs):
+        triggering_user = kwargs.get('user', None)
+        recipients = {}
+        if notification in ('approved', 'rejected'):
+            requested_by = workflow_state.requested_by
+            if requested_by != triggering_user:
+                recipients = {requested_by}
+        elif notification == 'submitted':
+            include_superusers = getattr(settings, 'WAGTAILADMIN_NOTIFICATION_INCLUDE_SUPERUSERS', True)
+            if include_superusers:
+                recipients = get_user_model().objects.filter(is_superuser=True)
+                if triggering_user:
+                    recipients.exclude(pk=triggering_user.pk)
```

Comment:
Is it intentional that this only happens inside the `if include_superusers` branch? It looks like that might have been changed inadvertently in ffcb331b21e2169706ccbf7cdb62778dc236a3b0.